Update rules for capitalizing domain objects. (#2027)
FusionAuth/fusionauth-site: site/docs/v1/tech/index.adoc

* *Premium Features*: FusionAuth has a full featured community edition, but some features are reserved for customer with a license key. This section outlines the features and how to use them.
* *APIs*: APIs are the heart of FusionAuth and are documented in depth in this section.
* *Release Notes*: The all important release notes so you can stay current on what is new in FusionAuth!

> * *Documentation Style Guidelines*: Here are some guidelines to follow when contributing documentation.

Reviewer: ```suggestion

== Documentation Conventions

```

> - Capitalize all domain objects, especially when calling the API in which the object is created and/or updated in FusionAuth.

Reviewer: ```suggestion
- We apitalize all domain objects, especially when calling the API in which the object is created and/or updated in FusionAuth.
```

For example, see the link:/docs/v1/tech/apis/api-keys[API Key API] description for `apiKeyId`, where API Key is capitalized: `The unique Id of the API Key to create. If not specified a secure random UUID will be generated.`

> - If referring to something that exists as a domain object in FusionAuth, but you are not explicitly referring to an object being created/updated in FusionAuth, use lowercase.

Reviewer: ```suggestion
- If referring to something which is a domain object in FusionAuth, but we are not explicitly referring to an object being created/updated in FusionAuth, we use lowercase.
```
